sdk/dapp-kit/src/index.ts

```diff
@@ -9,10 +9,11 @@ export * from './hooks/rpc/index.js';
 export * from './hooks/wallet/useCurrentAccount.js';
 export * from './hooks/wallet/useCurrentWallet.js';
 export * from './hooks/wallet/useWallets.js';
+export * from './hooks/wallet/useAccounts.js';
 export * from './hooks/wallet/useSwitchAccount.js';
 export * from './hooks/wallet/useConnectWallet.js';
 export * from './hooks/wallet/useDisconnectWallet.js';
 export * from './hooks/useSuiClientMutation.js';
 export * from './hooks/useSuiClientQuery.js';
 export * from './hooks/useSuiClientInfiniteQuery.js';
-// export * from './types.js';
```

Comment:
Oops, I messed up and accidentally landed this from my previous PR (was having an issue running tests) 😅